Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportExecutionLogger.cs

```diff
@@ -83,7 +84,9 @@ public async Task<bool> RunSetEnd(RunSetConfig runsetConfig, IExecutionLoggerMan
         public async Task<bool> SendRunsetEndDataToCentralDbTaskAsync(RunSetConfig runsetConfig)
         {
             AccountReportRunSet accountReportRunSet = AccountReportEntitiesDataMapping.MapRunsetEndData(runsetConfig);
-            return await AccountReportApiHandler.SendRunsetExecutionDataToCentralDBAsync(accountReportRunSet, true);
+            bool Response = await AccountReportApiHandler.SendRunsetExecutionDataToCentralDBAsync(accountReportRunSet, true);
+            AccountReportEntitiesDataMapping.accountReportStatistics = new List<AccountReportStatistics>();
```

Comment:
Reset it using by assigning it to just [] instead of new List<AccountReportStatistics>();

Reply:
Done.